autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/AutofillJavascriptInterface.kt

```diff
@@ -76,7 +78,11 @@ class AutofillStoredBackJavascriptInterface @Inject constructor(
 
     override var callback: Callback? = null
     override var webView: WebView? = null
+
+    // coroutine jobs tracked for supporting cancellation
     private val getAutofillDataJob = ConflatedJob()
+    private val storeFormDataJob = ConflatedJob()
```

Comment:
Yeah, it's used for cancellation. so if another `storeFormData` request came in when a previous one is still executing, that's unexpected. we could let them both execute, but figure because it's unexpected we should cancel any currently executing request.